### PR TITLE
Additonal HBW architecture

### PR DESCRIPTION
**Pull request by michalbiesek**
This would be useful for a fix related to MEMKIND_HBW_ALL - based on #491

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/492)

### CODE REVIEW

File: utils/qemu/topology/clx_2_var4_emul_hbw.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <sibling id='2' value='28'/>
+          <sibling id='3' value='10'/>
+        </distances>
+      </cell>
+      <interconnects>
+        <bandwidth initiator='0' target='0' type='read' value='204800' unit='MiB'/>
+        <bandwidth initiator='0' target='1' type='read' value='102400' unit='MiB'/>
+        <bandwidth initiator='0' target='2' type='read' value='409600' unit='MiB'/>
+        <bandwidth initiator='0' target='3' type='read' value='102400' unit='MiB'/>
+        <bandwidth initiator='1' target='1' type='read' value='204800' unit='MiB'/>
```
Comment (PatKamin) [inline utils/qemu/topology/clx_2_var4_emul_hbw.xml:51]: Missing values of bandwidth from node 0 to node 1.

**Review by PatKamin — COMMENTED** [review]
:lgtm:

> Reviewed 7 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/492#-:-MRxihnS8ab3GMYLlYxq:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @bratpiorka)

**Review by bratpiorka — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/492)* status: all files reviewed, 1 unresolved discussion (waiting on @michalbiesek)

___
*[utils/qemu/topology/README.md, line 22 at r1](https://reviewable.io/reviews/memkind/memkind/492#-MRxk2xR245pSYEtu4dO:-MRxk2xR245pSYEtu4dP:bn7htb8) ([raw file](https://github.com/memkind/memkind/blob/3057be0eee457c7d113cad423fb75cccef7b7a92/utils/qemu/topology/README.md#L22)):*
> ```Markdown
> | clx_2_var3_emul_hbw.xml | Cascade Lake 2 sockets* | DRAM + HBM    | X            | HBM on both sockets        |
> | clx_2_var3_hmat.xml     | Cascade Lake 2 sockets  | DRAM + PMEM   | X            | PMEM on both sockets       |
> | clx_2_var4_emul_hbw.xml | Cascade Lake 2 sockets* | HBM           | X            | HBM on all Nodes           |
> ```

why not DRAM + HBM? Also, we use "sockets" everywhere else

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/492)* status: all files reviewed, 1 unresolved discussion (waiting on @bratpiorka)

___
*[utils/qemu/topology/clx_2_var4_emul_hbw.xml, line 51 at r1](https://reviewable.io/reviews/memkind/memkind/492#-MRxcWZMLzUaCILLrEb4-r1-51:-MRxkkdT0IhQDj_QZP-A:b-xmqqn2) ([raw file](https://github.com/memkind/memkind/blob/3057be0eee457c7d113cad423fb75cccef7b7a92/utils/qemu/topology/clx_2_var4_emul_hbw.xml#L51)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

Missing values of bandwidth from node 0 to node 1.
</blockquote></details>

This is a limitation of libvirt I will try to prepare a fix for this
___
*[utils/qemu/topology/README.md, line 22 at r1](https://reviewable.io/reviews/memkind/memkind/492#-MRxk2xR245pSYEtu4dO:-MRxkpj52P8qZo0n6dDg:b-eogt02) ([raw file](https://github.com/memkind/memkind/blob/3057be0eee457c7d113cad423fb75cccef7b7a92/utils/qemu/topology/README.md#L22)):*
<details><summary><i>Previously, bratpiorka (Rafał Rudnicki) wrote…</i></summary><blockquote>

why not DRAM + HBM? Also, we use "sockets" everywhere else
</blockquote></details>

Because I want all Nodes to be treated as High Bandwidth Memory - even these with CPU. I put all Nodes because all Nodes are treated as HBW, see commit message for details. If I would use sockets there would be no distinction between clx_2_var4_emul_hbw.xml and clx_2_var3_emul_hbw.xml

**Review by bratpiorka — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/492#-:-MRz0s2N-gU-9rAP6F4n:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @michalbiesek)

**Pull request merged**